crates/bevy_internal/src/default_plugins.rs

```diff
@@ -56,6 +56,12 @@ plugin_group! {
         bevy_gizmos:::GizmoPlugin,
         #[cfg(feature = "bevy_state")]
         bevy_state::app:::StatesPlugin,
+        #[cfg(feature = "bevy_picking")]
```

Comment:
Plugin tuples can nest arbitrarily right? Just a nitpick (feel free to ignore), would be more concise if you throw these three into a tuple (or even a plugin group) and put the `#[cfg]` just once.

Reply:
I fixed it into one single plugin, please resolve if you think this is enought.